app/.gitignore

```diff
@@ -0,0 +1,82 @@
+.DS_Store
```

Comment:
Hi @leonardnjura 
Only `ignore` the **/build** folder in the app level `.gitignore` file.